cores/rp2040/RP2040.h

```diff
@@ -44,48 +47,64 @@ class _MFIFO {
         _multicore = true;
     }
 
-    void registerCore() {
+    void registerCore()
+    {
         multicore_fifo_clear_irq();
         irq_set_exclusive_handler(SIO_IRQ_PROC0 + get_core_num(), _irq);
         irq_set_enabled(SIO_IRQ_PROC0 + get_core_num(), true);
     }
 
-    void push(uint32_t val) {
-        while (!push_nb(val)) { /* noop */ }
+    void push(uint32_t val)
+    {
+        while (!push_nb(val))
+        { /* noop */
+        }
     }
 
-    bool push_nb(uint32_t val) {
+    bool push_nb(uint32_t val)
+    {
         // Push to the other FIFO
         return queue_try_add(&_queue[get_core_num() ^ 1], &val);
     }
 
-    uint32_t pop() {
+    uint32_t pop()
+    {
         uint32_t ret;
-        while (!pop_nb(&ret)) { /* noop */ }
+        while (!pop_nb(&ret))
+        { /* noop */
+        }
         return ret;
     }
 
-    bool pop_nb(uint32_t *val) {
+    bool pop_nb(uint32_t *val)
+    {
         // Pop from my own FIFO
         return queue_try_remove(&_queue[get_core_num()], val);
     }
 
-    int available() {
+    int available()
```

Comment:
Can you please undo the formatting-only changes?  That breaks the CI style checks.

Reply:
Sure!